onnxruntime/core/providers/nuphar/common/nuphar_tvm_utils.cc

```diff
@@ -156,7 +154,7 @@ void SaveTVMModuleToCache(const std::string& filename, tvm::runtime::Module& mod
   if (existing_files.count(filename) == 0 &&
       GetOrCreateTVMModuleCacheDirectory(path, /*create*/ true)) {
     existing_files.insert(filename);
-    path.append("cached_" + std::to_string(saved_tvm_model_cnt++) + ".o");
+    path.append(filename + ".o");
```

Comment:
Any particular reason to change the existing behavior? We use 'cached_' + cnt instead of filename, because filename may contain model-sensitive information.

Reply:
The filename comes from GetPackedFuncName below, which uses subgraph.UniqueId() and no longer carry node info about the subgraph.

---
In reply to: [352328327](https://github.com/microsoft/onnxruntime/pull/2514#discussion_r352328327) [](ancestors = 352328327)